routes/index.js

```diff
@@ -2,11 +2,19 @@ const express = require('express');
 // eslint-disable-next-line new-cap
 const router = express.Router();
 const path = require('path');
+const axios = require('axios');
```

Comment:
We prefer to use the `node-fetch` package. This way it is more in sync with the browsers `fetch` API

Reply:
This won't be a blocker just an FYI

Reply:
Yea that is fine just letting you know going forward to try and use the `node-fetch`